doc/cookbook/source/references.bib

```diff
@@ -153,3 +153,15 @@ @article{Quinonero-Candela2005
  year = {2005},
  pages = {1939--1959},
 }
+@TECHREPORT{Wickelmaier03anintroduction,
+    author = {Florian Wickelmaier},
+    title = {An introduction to MDS},
+    institution = {},
+    year = {2003}
+}
+@inproceedings{Silva2004SparseMS,
+  title={Sparse multidimensional scaling using landmark points},
+  author={Vin de Silva and Joshua B. Tenenbaum},
+  booktitle={},
```

Comment:
watch you bibtex style, this needs a conference name (check the others to see how we name them)